pandas/core/frame.py

```diff
@@ -4764,36 +4764,49 @@ def _gotitem(self, key, ndim, subset=None):
         return self[key]
 
     _agg_doc = dedent("""
+    Notes
+    -----
+    The default behavior of aggregating over the axis 0 is different from
+    `numpy` functions `mean`/`median`/`prod`/`sum`/`std`/`var`, where the
```

Comment:
Does sphinx complain about having a character immediately after the backticks? May need spaces.

Reply:
I have added commas instead.